deploy/olm-catalog/cluster-network-addons/0.0.1/cluster-network-addons-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,289 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: kubevirt-cluster-network-addons-operator.v0.0.1
+  namespace: placeholder
```

Comment:
what it will be replaced with?

Reply:
`placeholder` is something OLM fills in when a subscription is made.